asteroid/filterbanks/transforms.py

```diff
@@ -47,11 +47,18 @@ def mul_c(inp, other, dim: int = -2):
     return torch.cat([real1 * real2 - imag1 * imag2, real1 * imag2 + imag1 * real2], dim=dim)
 
 
-def take_reim(x, dim: int = -2):
-    return x
+def reim(x, dim: int = -2) -> Tuple[torch.Tensor]:
```

Comment:
Shouldn't this be `Tuple[torch.Tensor, torch.Tensor]`

Reply:
I'm not very familiar with typing yet, you're right, I'll fix it.

Reply:
And now the linter is complaining.. 
```def chunk(self: Tensor, chunks: _int, dim: _int=0) -> Union[Tuple[Tensor, ...], List[Tensor]]: ...```